### ISSUE TITLE
very bad result in the real-plate images

### ISSUE THREAD

**Issue by mzeynali** [qjadud1994/CRNN-Keras#24]
Hi,
I've trained the VGG-GRU network for around 700k artificially generated images, and i got 96% accuracy for test data (about  140k generated image), but when i test the trained the model on real-images , the results are very bad, why ?

**Comment by qjadud1994** [comment]
Due to overfitting, it is no wonder that it does not work properly in real images.

so, I recommend pre-training with synthetic images and fine-tune with real data.

**Comment by danishansari** [comment]
How would you recommend fine-tuning, should we use differential learning rates, i.e. lower lr for CNN and higher for RNN? Or a global lower learning rate like 1e-3.

**Comment by zeynali** [comment]
@qjadud1994, Because I used Early Stopping callback for 3 epoch, in my opinion, Should not happen the overfitting problem, right?
